cmd/snap/cmd_run.go

```diff
@@ -193,6 +193,29 @@ func snapRunHook(snapName, snapRevision, hookName string) error {
 	return runSnapConfine(info, hook.SecurityTag(), snapName, "", hook.Name, nil)
 }
 
+// FIXME: move to osutil?
+func envMap(env []string) map[string]string {
+	envMap := map[string]string{}
```

Comment:
did you intend to `make(map[string]string{}` here? (not sure how go behaves ATM)